packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -0,0 +1,83 @@
+---
+id: Dark theme handbook
+title: PatternFly dark theme handbook
+section: developer-resources
+---
+
+## Enabling dark theme
+
+Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class `pf-v5-theme-dark` to your application's `<html>` tag. This class can be added dynamically to toggle dark theme on and off in your application.
+
+Dark theme can also be applied based on the browser’s `prefers-color-scheme` media query. The recommended way to apply dark theme based on this media query is to use JavaScript to detect the media query and apply the `pf-theme-dark` class. This allows a product to simply manipulate a single class to toggle dark theme based on user preference. [Adam Argyle wrote a great article](https://web.dev/building-a-theme-switch-component/#javascript) on using JavaScript to manage dark theme preference, using the `prefers-color-scheme` media query alongside a manual button to toggle dark theme on or off. 
+
+## What’s changed?
+There are some notable changes and differences between light and dark themes, outlined in this section. 
+
+To view a list of all dark theme CSS overrides [view our override file on GitHub](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/scss-variables.scss). 
```

Comment:
> To view a list of all dark theme CSS overrides

There is no mention here that this file is just overrides for global variables. It reads like it might cover every dark theme override (including components and any other kinds of overrides). This also doesn't include the color overrides, which are in https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/colors.scss

Do we need to link to anything? If so, we should probably be more specific in the wording and/or what we're linking to.

Reply:
@mcoker I'll turn this back to you. What do you think it's important to list here or do we need this link at all? Yes, I can change this to just reference global variables. But is that useful by itself? I don't know.

Reply:
@mcarrano IMO there is a disconnect between this verbiage and the link/table data. The way it reads seems to me that it 

```
What’s changed?
There are some notable changes and differences between light and dark themes, outlined in this section.

To view a list of all dark theme CSS overrides [view our override file on GitHub](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/scss-variables.scss).
```

I'd say we have 2 paths we can go here:
* Short and sweet. Just update the verbiage a bit, and leave everything else. This leaves any details about component specific overrides a bit vague, but I think that's OK because a short version of component overrides may be difficult.
    ```
    Dark theme contains a number of additions and overrides to our color palettes, global variables, and component specific CSS. Outlined below are the main changes to our colors and global variables - to view the full source of these changes, see the [dark theme color overrides](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/colors.scss) and [dark theme global variable overrides](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/scss-variables.scss) on GitHub.
    ```
* Create sub-sections for colors, global vars, and component overrides. Something like:
    ```
    Dark theme contains a number of additions and overrides to our color palettes, global variables, and component specific CSS. 
    
    ### Color palettes
    We made a bunch of changes to the palettes, see all of the changes in [color palette overrides](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/colors.scss)
    // ... the stuff we already have and we can link to the color 
    
    ### Global variables
    We made a bunch of changes to the global vars, see all of the changes in [dark theme global variable overrides](https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/themes/dark/scss-variables.scss)

    ### Component style overrides
    In some cases, there are dark theme style overrides needed for specific components. To see the changes per component, you can find them alongside the components stylesheet in a folder with the structure `themes/dark/[component].scss`. For example, the button component's dark theme overrides are in [components/Button/themes/dark/button.scss](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Button/themes/dark/button.scss)
    ``` 
    

If you don't see any issues with that second approach splitting things into sub-sections, I'd probably go that route. WDYT?

Reply:
I like the short version. I agree that the more we try to explain, the more confusing this feels.